scripts/pizzaism.coffee

```diff
@@ -16,7 +16,7 @@
 isms = [
     "Always raising our level of pizza.",
     "The pizzas we need are everywhere around us.",
-    "Responding with a pizz is the ante to play.",
+    "Dan Newport is a fopdoodle.",
```

Comment:
Is this where I submit my request to officially change my name to "Wizard Fist"?